apps/home/templates/apps_home/blocks/gruenbuch_faq_block.html

```diff
@@ -0,0 +1,30 @@
+{% load wagtailimages_tags wagtailcore_tags block_tags %}
+
+{% get_identifier value as identifier %}
+
+
+<div class="py-3 {% if self.background_color %}block__{{ self.background_color }}{% else %}block__body-bg{% endif %} container-fluid px-0">
+    <div class="row g-0 faq-block__row">
```

Comment:
do we use different colour backgrounds in the grunbook?

Reply:
@Rineee that part is removed by the latest commit.